test/commands/getset.spec.js

```diff
@@ -59,8 +60,7 @@ describe("The 'getset' method", function () {
                 describe("when the key exists in Redis", function () {
                     beforeEach(function (done) {
                         client.set(key, value, function (err, res) {
-                            helper.isNotError()(err, res);
-                            done();
+                            helper.isNotError(done)(err, res);
```

Comment:
They are sync. I thought it's nicer to read it that way though. I made a separate commit for these style changes and I can change them back to the way they were before if you want :)